api_app/analyzers_manager/migrations/0064_analyzer_config_phoneinfoga.py

```diff
@@ -0,0 +1,136 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "module": "phoneinfoga_scan.Phoneinfoga",
+        "base_path": "api_app.analyzers_manager.observable_analyzers",
+    },
+    "name": "Phoneinfoga",
+    "description": "[Phoneinfoga](https://sundowndev.github.io/phoneinfoga/)",
+    "disabled": False,
+    "soft_time_limit": 60,
+    "routing_key": "default",
+    "health_check_status": True,
+    "type": "observable",
+    "docker_based": True,
+    "maximum_tlp": "GREEN",
+    "observable_supported": ["generic"],
+    "supported_filetypes": [],
+    "run_hash": False,
+    "run_hash_type": "",
+    "not_supported_filetypes": [],
+    "health_check_task": None,
+    "model": "analyzers_manager.AnalyzerConfig",
+}
+
+params = [
+    {
+        "python_module": {
+            "module": "phoneinfoga_scan.Phoneinfoga",
+            "base_path": "api_app.analyzers_manager.observable_analyzers",
+        },
+        "name": "scanner_name",
+        "type": "str",
+        "description": "scanner name for phoneinfoga",
```

Comment:
please provide link to the official doc so the people can find it and choose the scanner type easier